src/main/java/com/aspose/imaging/cloud/sdk/api/ImagingApi.java

```diff
@@ -248,7 +213,7 @@ else if (File.class.getName().equals(SearchResultsSet.class.getName()))
      * @return File
      * @throws Exception 
      */
-    public ApiResponse deleteSearchContextImage(DeleteSearchContextImageRequest request) throws Exception 
+    public byte[] deleteSearchContextImage(DeleteSearchContextImageRequest request) throws Exception 
     {
```

Comment:
Why do we need this change ?

Reply:
@aifeigin I consider it the best change of this PR :) Due to default File response in the code generated by Swagger Codegen, we needed either to write responses in files and return them to users (which is a bad option in any perspective), or add additional code complexity to keep in-memory processing (which has been done as not ideal, but still better solution). During Node.js SDK implementation, I've found out how to add _vendor extensions_ (in other words, custom data) for Swagger and consume it in Codegen templates to generate better code, which made possible to remove the complexity. Please, see changed examples to understand how it affected users before https://github.com/aspose-imaging-cloud/aspose-imaging-cloud-java/pull/15/files#diff-27b714cfcd9ee7241d8db7efa832b94a

Reply:
1. That's right, though considering current product state I think it's better to introduce this change earlier to reduce the need of manual code changes when the model changes each time.
2. Error codes are handled other way, please see https://github.com/aspose-imaging-cloud/aspose-imaging-cloud-java/blob/master/src/main/java/com/aspose/imaging/cloud/sdk/invoker/internal/requesthandlers/ApiExceptionRequestHandler.java
3. .NET generator doesn't have this problem, so in fact Java finally goes .NET way.

Reply:
_Error codes are handled other way_
Please, illustrate actual error handling approach in the API usage examples.

Reply:
@aifeigin OK, I will add this to examples: basically it will be simple _try_ with _catch_ of _ApiException_.